lib/rails_response_dumper/runner.rb

```diff
@@ -119,6 +117,12 @@ def run_dumps
             dump[:timestamp] = timestamp.iso8601 unless options[:exclude_timestamp]
 
             File.write("#{dumper_dir}/#{index}.json", JSON.pretty_generate(dump))
+            File.open("#{dumper_dir}/#{index}.request_body", 'w', external_encoding: 'ASCII-8BIT') do |f|
```

Comment:
You can use `File.write` for a shorter, more concise syntax:

The body is binary data and shouldn't go through a text encoding.

I also think we should drop "_body" from these file names as I don't find it adds any context but it does add general noise to the directory. 

Same for the response.

```suggestion
            File.write("#{dumper_dir}/#{index}.request", request.body.string, mode: 'wb')
```